### PR TITLE
KEYCLOAK-13908: Database Migration to >=9.0.1 fails on MySQL

### PR DESCRIPTION
**Pull request by rmartinc**
Documentation fix for KEYCLOAK-13908. Finally it was decided that no coding action will be added and only a documentation warning will be added.
Feel free to change anything. I spend a lot of time with these doc PRs and I'm never totally happy with them. :-)

### CODE REVIEW

**Review by mposolda — DISMISSED** [review]
@rmartinc Thanks! I am adding also Andy as a reviewer of this PR - he is our documentation expert.

@andymunro Do you have a chance to review this please?

File: upgrading/topics/keycloak/changes.adoc [hunk 1]
```diff
@@ -8,6 +8,21 @@ It is no longer necessary to set promiseType in the JavaScript adapter, and both
 recommended to update applications to use native promise API (`then` and `catch`) as soon as possible, as the legacy API
 (`success` and `error`) will be removed at some point.
 
+==== Duplicated top level groups
+
+Version 9.0.1 fixes a problem which could create duplicated top level groups in the realm. Nevertheless the existance
```
Comment (andymunro) [inline upgrading/topics/keycloak/changes.adoc]: existence

File: upgrading/topics/keycloak/changes.adoc [hunk 2]
```diff
@@ -8,6 +8,21 @@ It is no longer necessary to set promiseType in the JavaScript adapter, and both
...
+of previous duplicated groups makes the upgrade process fail. The {project_name} server can be affected by this issue
+if it is using a H2, MariaDB, MySQL or PostgreSQL database. Before launching the upgrade check if the server contains
```
Comment (andymunro) [inline upgrading/topics/keycloak/changes.adoc]: comma after upgrade

File: upgrading/topics/keycloak/changes.adoc [hunk 3]
```diff
@@ -8,6 +8,21 @@ It is no longer necessary to set promiseType in the JavaScript adapter, and both
...
+duplicated top level groups, for example the following SQL query can be executed at database level to list them:
```
Comment (andymunro) [inline upgrading/topics/keycloak/changes.adoc]: make "for example..."  a new sentence.

**Review by andymunro — CHANGES_REQUESTED** [review]
A few changes.

**Comment by rmartinc** [comment]
> A few changes.

Thanks @andymunro! The three changes you requested are incorporated....

**Review by mposolda — APPROVED** [review]

**Pull request merged**